icons/qubesos.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Qubes OS</title><path d="M11.92 0a2.4 2.405 0 00-.063 0 2.4 2.405 0 00-.042.002 2.4 2.405 0 00-.001 0 2.4 2.405 0 00-.003 0 2.4 2.405 0 00-.041.002 2.4 2.405 0 00-.038.003 2.4 2.405 0 00-.005 0 2.4 2.405 0 00-.024.002 2.4 2.405 0 00-.043.004 2.4 2.405 0 00-.041.005 2.4 2.405 0 00-.004 0 2.4 2.405 0 00-.041.006 2.4 2.405 0 00-.043.006 2.4 2.405 0 00-.043.007 2.4 2.405 0 00-.128.027 2.4 2.405 0 00-.651.257l-5.408 3.13-2.7 1.562A2.4 2.405 0 001.4 7.096v9.384a2.4 2.405 0 001.2 2.08l8.108 4.693a2.4 2.405 0 001.202.323 2.4 2.405 0 00.002 0 2.4 2.405 0 00.055-.001 2.4 2.405 0 00.076-.003 2.4 2.405 0 001.065-.317l1.803-1.044 2.302 1.34c1.029.599 2.686.599 3.716 0l1.029-.6c1.029-.598 1.029-1.563 0-2.162l-2.29-1.332 1.547-.895a2.4 2.405 0 001.2-2.05 2.4 2.405 0 000-.002 2.4 2.405 0 000-.028V7.099a2.4 2.405 0 000-.003 2.4 2.405 0 000-.04 2.4 2.405 0 000-.004 2.4 2.405 0 00-.002-.04 2.4 2.405 0 00-1.198-2l-3.58-2.07-4.529-2.62a2.4 2.405 0 00-.434-.196 2.4 2.405 0 00-.083-.027 2.4 2.405 0 00-.001 0 2.4 2.405 0 00-.04-.012 2.4 2.405 0 00-.002 0 2.4 2.405 0 00-.04-.01 2.4 2.405 0 00-.002-.001 2.4 2.405 0 00-.041-.01 2.4 2.405 0 00-.002 0 2.4 2.405 0 00-.04-.01 2.4 2.405 0 00-.002 0 2.4 2.405 0 00-.04-.01 2.4 2.405 0 00-.002 0 2.4 2.405 0 00-.042-.007 2.4 2.405 0 00-.001 0 2.4 2.405 0 00-.042-.008 2.4 2.405 0 00-.001 0 2.4 2.405 0 00-.085-.012 2.4 2.405 0 00-.001 0 2.4 2.405 0 00-.002 0 2.4 2.405 0 00-.128-.013 2.4 2.405 0 00-.11-.005 2.4 2.405 0 00-.02 0A2.4 2.405 0 0011.92 0zm-.024 5.305a1.32 1.323 0 01.288.029 1.32 1.323 0 01.032.007 1.32 1.323 0 01.031.008 1.32 1.323 0 01.031.009 1.32 1.323 0 01.031.01 1.32 1.323 0 01.061.02 1.32 1.323 0 01.06.025 1.32 1.323 0 01.03.013 1.32 1.323 0 01.023.01 1.32 1.323 0 01.006.003 1.32 1.323 0 01.008.005 1.32 1.323 0 01.02.01 1.32 1.323 0 01.009.005 1.32 1.323 0 01.02.01 1.32 1.323 0 01.008.005 1.32 1.323 0 01.014.008L15.367 7.1l1.66.96a1.32 1.323 0 01.005.003 1.32 1.323 0 01.023.014 1.32 1.323 0 01.027.017 1.32 1.323 0 01.426.45v-.002a1.32 1.323 0 01.003.004l-.003.002a1.32 1.323 0 01.178.661v5.16a1.32 1.323 0 01-.056.381 1.32 1.323 0 01-.001.002 1.32 1.323 0 01-.01.029 1.32 1.323 0 010 .002 1.32 1.323 0 01-.01.029 1.32 1.323 0 010 .001 1.32 1.323 0 01-.01.029 1.32 1.323 0 01-.001.002 1.32 1.323 0 01-.571.667l-4.46 2.582a1.32 1.323 0 01-1.319 0l-4.46-2.582a1.32 1.323 0 01-.66-1.143V9.207a1.32 1.323 0 01.178-.662l.002.001a1.32 1.323 0 01.482-.484l4.46-2.58a1.32 1.323 0 01.646-.177z"/></svg>
```

Comment:
Hello
Thanks for the PR!
Also have a close look into our [CONTRIBUTING guidlines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md) and here to our standards. (the svg is not what the linter expects, thus the build is failing currently) (icon name and icon position)

Reply:
@d3ward, I've been able to regenerate the path so that it passes the linter properly. Feel free to use this one, or regenerate your own!

```svg
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Quebes OS icon</title><path d="M21.893 20.79l-2.289-1.332 1.547-.895a2.402 2.402 0 0 0 1.2-2.08V7.098l-.003-.059a2.402 2.402 0 0 0-1.198-2.027l-3.899-2.255-4.21-2.436a2.473 2.473 0 0 0-.237-.118L12.77.187l-.093-.036-.052-.019c-.028-.01-.057-.018-.085-.027l-.062-.019-.079-.02-.072-.017-.073-.013-.079-.013-.068-.008-.087-.008-.063-.004A10.324 10.324 0 0 0 11.9 0h-.03l-.082.001-.076.002-.093.007-.064.006c-.037.004-.073.01-.11.016-.014.004-.029.006-.044.009a3.266 3.266 0 0 0-.154.034 2.39 2.39 0 0 0-.602.245L5.536 3.277l-3 1.736a2.407 2.407 0 0 0-1.201 2.083v9.385a2.405 2.405 0 0 0 1.2 2.08l8.108 4.693a2.395 2.395 0 0 0 2.4.002l1.804-1.044 2.302 1.339c1.03.599 2.687.599 3.716 0l1.03-.6c1.027-.597 1.027-1.562-.002-2.161zm-10.71-2.695l-4.46-2.583a1.324 1.324 0 0 1-.66-1.143V9.206c0-.236.063-.464.177-.662l.002.001c.116-.2.282-.368.482-.485l4.459-2.58c.092-.053.189-.093.289-.122l.034-.01c.035-.01.07-.015.105-.022.023-.004.045-.01.068-.013.031-.004.062-.004.093-.006.028 0 .055-.004.083-.004.036 0 .073.004.11.007.02.002.038.002.058.005.037.005.074.014.11.022.018.004.037.007.055.012.04.011.077.025.115.04l.045.015c.052.022.104.047.154.076l4.46 2.58c.198.116.364.283.48.483l.002.003-.003.002c.116.201.177.43.177.661v5.161c0 .15-.028.295-.076.433a1.32 1.32 0 0 1-.583.71l-4.46 2.582a1.312 1.312 0 0 1-1.316 0z"/></svg>
```

Reply:
Thank you for your comments and help
Sorry for the delay 

P.S. Happy New Year !